### PR TITLE
Fix changelog

### PR DESCRIPTION
**Pull request by cdce8p**
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :scroll: Docs          |

## Related Issue
#4422

### CODE REVIEW

**Review by Pierre-Sassoulas — COMMENTED** [review]
Didn't we change that for releasing often and not bothering having a bug fixing branch ? I know only you and hippo participated to the discussion, but we're also the one doing the actual release work.

**Comment by cdce8p** [comment]
> Didn't we change that for releasing often and not bothering having a bug fixing branch ? I know only you and hippo participated to the discussion, but we're also the one doing the actual release work.

AFAIK we didn't reached a final conclusion yet. The main issue here is that the entry should belong to `2.9.0` instead of the already released `2.8.1`.

Btw, don't know why the tests are failing all of the sudden.

**Comment by Pierre-Sassoulas** [comment]
The release.md was modified in #4263.

Yeap, it's strange, the latest commit on master passed.

**Comment by cdce8p** [comment]
> Yeap, it's strange, the latest commit on master passed.

I managed to narrow it down a bit. It seems like the last Python update caused it (`3.8.10` and `3.9.5`). I just checked `3.9.4` and the tests pass. Just from the changelog I would guess it's this change although I haven't read it yet: https://bugs.python.org/issue43105

We probably need to update our tests then 🤷🏻‍♂️

File: ChangeLog [hunk 1]
```diff
@@ -2,15 +2,21 @@
...
 ..
   Put new features and bugfixes here and also in 'doc/whatsnew/2.9.rst'
 
+* The warning for ``arguments-differ`` now signals explicitly the difference it detected
+  by naming the argument or arguments that changed and the type of change that occurred.
+
+
+What's New in Pylint 2.8.3?
+===========================
+Release date: TBA
```
Comment (cdce8p) [inline ChangeLog:18]: @Pierre-Sassoulas Should I remove the `2.8.3` section?

Comment (Pierre-Sassoulas) [reply ChangeLog:18]: I think we're going to release a 2.8.3 next. It depends on how long the blocker takes ?

**Pull request merged**